### PR TITLE
#51 allow user to enter away mode temperature preferences for summer and winter

### PR DESCRIPTION
**Pull request by rileykc98**

### CODE REVIEW

**Review by mandocino — APPROVED** [review]
Great work!

**Review by loganmcmullen — APPROVED** [review]
Code looks clean, gj

File: src/com/simulator/view/Dashboard.fxml [hunk 1]
```diff
@@ -24,7 +30,7 @@
 <?import javafx.scene.layout.VBox?>
 <?import javafx.scene.text.Font?>
 
-<VBox prefHeight="583.0" prefWidth="1234.0" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.simulator.controller.SmartHomeSimulatorController">
+<VBox prefHeight="583.0" prefWidth="1234.0" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.simulator.controller.SmartHomeSimulatorController">
```
Comment (bowe99) [inline src/com/simulator/view/Dashboard.fxml:33]:
```suggestion
<VBox prefHeight="583.0" prefWidth="1234.0" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.simulator.controller.SmartHomeSimulatorController">
```

**Review by bowe99 — APPROVED** [review]
Looks great Riley good work

**Pull request merged**